src/domUpdates.js

```diff
@@ -1,23 +1,57 @@
-const pastBookingsUl = document.querySelector(".past-bookings-details");
-const currentBookingsUl = document.querySelector(".current-bookings-details");
-const futureBookingsUl = document.querySelector(".future-bookings-details");
+const pastBookingsTable = document.querySelector(".past-bookings-details");
+const currentBookingsTable = document.querySelector(
+  ".current-bookings-details"
+);
+const futureBookingsTable = document.querySelector(".future-bookings-details");
 
-const createBookingCard = (booking) => {
-  const li = document.createElement("li");
-  li.innerText = `Date Booked: ${booking.date} : Room Number: ${booking.roomNumber}`;
-  return li;
+const createTr = (columns, columnType = "td") => {
+  const tr = document.createElement("tr");
+  columns.forEach((column) => {
+    const td = document.createElement(columnType);
```

Comment:
I would consider not using abbreviations just for easier reading. Code looks great though.

Reply:
Nice work; although it's OK to have the coloumnType assigned to "td" as a parameter. I prefer to have that assignment inside a function rather than a parameter. Either way, it works. 
Abbreviations are not recommended since we want code reviews to focus on issues more important than syntax and naming standards.